### PR TITLE
Migrations and models

### LINKED ISSUE #12: Migrate Users Table

### PR DESCRIPTION
**Pull request by the-color-bliu**
## Types of Changes
Migrate user, track, verse, vote tables, create and test models for each of them as well

- [x] This PR closes #12 #6 #5 #4 #13 

- [ ] Bug fix (non-breaking change which fixes an issue): 

- [x] New feature (non-breaking change which adds functionality):

- [ ] Breaking change (fix or feature that would cause existing functionality to change):

- [ ] My change requires a change to the documentation.

- [ ] I have updated the documentation accordingly.

- [x] I have added tests to cover my changes.

- [x] All new and existing tests passed.

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -33,3 +33,12 @@ class Application < Rails::Application
     config.api_only = true
   end
 end
+
```
Comment (ETBassist) [inline config/application.rb:36]: Oh this is interesting - is this there to solve a problem, or to prevent us from accidentally adding these bits in?

**Review by ETBassist — APPROVED** [review]
Looks good to me! Added a question about the commented out bit in `config/application.rb` just because I've never seen it before and I'm curious what it does
Happy Patrick

File: config/application.rb (hunk 1 above)
Comment (SageOfCode) [reply config/application.rb:36]: Good question, I saw something similar to this in the GraphQL implementation walk through.

**Review by SageOfCode — APPROVED** [review]
Looks great to me, I'm looking forward to getting these tables on graphql

**Pull request merged**